java-api-core/src/com/kinvey/java/network/NetworkManager.java

```diff
@@ -679,7 +678,7 @@ public Aggregate aggregate(ArrayList<String> fields, AggregateEntity.AggregateTy
         Get(String queryString, Class myClass){
         	super(client, "GET", REST_PATH, null, myClass);
         	this.collectionName= NetworkManager.this.collectionName;
-        	this.queryFilter = queryString;
+        	this.queryFilter = !queryString.equals("{}") ? queryString : null;
```

Comment:
@yuliya-guseva Is it possible here that `queryString` can be null?  If so, we should guard against that case.